Slim/Networking/SimpleAsyncWS.pm

```diff
@@ -0,0 +1,396 @@
+package Slim::Networking::SimpleAsyncWS;
+
+# Lyrion Music Server Copyright 2024 Lyrion Community.
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License,
+# version 2.
+
+# This class provides a non-blocking WebSockets client connection from Lyrion Music Server.
+
+# This class is intended for plugins and other code needing simply to
+# handle a persistent websockets connection.  If you have more complex
+# needs consider writing a fuller implementation.
+
+# more documentation at end of file.
+
+use strict;
+
+use IO::Socket;
+use IO::Socket::SSL;
+use IO::Select;
+use Protocol::WebSocket::Client;
+
+use Data::Dumper;
```

Comment:
Please use `Data::Dump` instead, as we're not using `Data::Dumper` anywhere else.

Reply:
Sure, no problem, I'll remove that.